src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddDebugLogProcessorPass.php

```diff
@@ -30,6 +30,14 @@ public function process(ContainerBuilder $container)
         }
 
         $definition = $container->getDefinition('monolog.logger_prototype');
+        $definition->setConfigurator([__CLASS__, 'configureLogger']);
         $definition->addMethodCall('pushProcessor', [new Reference('debug.log_processor')]);
     }
+
+    public static function configureLogger($logger)
+    {
+        if (method_exists($logger, 'removeDebugLogger') && 'cli' === \PHP_SAPI) {
```

Comment:
what about the phpdbg SAPI ? It is also a CLI one

Reply:
Fixed, thanks